### PR TITLE
Add a BERT example using Intel Neural Compressor SmoothQuant

### PR DESCRIPTION
**Pull request by yuwenzho**
## Describe your changes

Add a BERT example using Intel® Neural Compressor SmoothQuant.
Add an introduction of Intel® Neural Compressor SmoothQuant to the README.

## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [x] Update documents if necessary.
- [x] Format your code by running `pre-commit run --all-files`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.

### CODE REVIEW

**Comment by guotuofeng** [comment]
do we need enable it in CI pipeline?

**Comment by yuwenzho** [comment]
> do we need enable it in CI pipeline?

Yes, we do need to enable it in the CI pipeline. Could you please provide some guidance or suggestions on how to do this? It would be of great help.

**Comment by yuwenzho** [comment]
I enabled `bert_inc_ptq_cpu` and `bert_inc_smoothquant_ptq_cpu` in CI pipeline

File: examples/test/test_bert_ptq_cpu.py [hunk 1]
```diff
@@ -23,12 +23,15 @@ def setup():
 @pytest.mark.parametrize("search_algorithm", ["tpe"])
 @pytest.mark.parametrize("execution_order", ["joint"])
 @pytest.mark.parametrize("system", ["local_system", "aml_system", "docker_system"])
-@pytest.mark.parametrize("olive_json", ["bert_ptq_cpu.json"])
+@pytest.mark.parametrize("olive_json", ["bert_ptq_cpu.json", "bert_inc_ptq_cpu.json", "bert_inc_smoothquant_ptq_cpu.json"])
```
Comment (trajepl) [inline examples/test/test_bert_ptq_cpu.py:26]: Emm, if so, current Bert pipeline will run with about 3x times than before(about 1h). As it is only test the inc quantizaion passes, maybe we do not need to run the complete E2E tests?

I believe INC team already had their tests but not in Olive just like other ONNX passes.

**Review by trajepl — APPROVED** [review]

File: examples/test/test_bert_ptq_cpu.py (hunk 1 above)
Comment (yuwenzho) [reply examples/test/test_bert_ptq_cpu.py]: Thanks for your suggestion. I removed Bert INC test from the CI pipeline. All INC-related examples are tested internally by INC team.

**Review by trajepl — APPROVED** [review]

**Pull request merged**